hack/devicedb.conf.tpl.yaml

```diff
@@ -21,6 +21,7 @@ history:
    forwardInterval: 60000
    forwardThreshold: 100
    forwardBatchSize: 1000
+#nodeid: 39c54e945e3d11ea8c3a0242ac120002
```

Comment:
Can this be removed?

Reply:
I just provided this as an example. Because the nodeid field has been added and is mandatory if TLS certificates are not provided.

Reply:
Might be better to just remove this. Since we don't really want to specify any value for a node id here. Or we could just set that to refer another env variable or something like that.

Reply:
Might be better to just remove this. Since we don't really want to specify any value for a node id here. Or we could just set that to refer another env variable or something like that.

Reply:
It is commented out. It is just for reference.

Reply:
Ya no more dead code in our codebase. We need to document all the available config options somewhere else.

Reply:
Okay. Removing it.